docs/book/src/getting-started/testdata/project/cmd/main.go

```diff
@@ -55,16 +56,13 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
-	var secureMetrics bool
 	var enableHTTP2 bool
-	flag.StringVar(&metricsAddr, "metrics-bind-address", "0", "The address the metric endpoint binds to. "+
-		"Use the port :8080. If not set, it will be 0 in order to disable the metrics server")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8443", "The address the metric endpoint binds to. "+
+		"Use 0 in order to disable the metrics server")
```

Comment:
```suggestion
flag.StringVar(&metricsAddr, "metrics-bind-address", "0",  
	"The address the metrics endpoint binds to. " +
	"Use :8443 for HTTPS with metrics-secure set to true, " +
	"Use :8080 for HTTP with metrics-secure set to false, " +
	"or leave as 0 to disable the metrics service.")
```

How about we keep the metrics service  as 0/disabled as default set? We can set up the kustomize scaffold to enable it, but leave it uncommented so that we have the metrics enabled but we give a fair chance for users easily disable/opt out  if they wish to do so.